lib/tasks/add_navigation_links.rake

```diff
@@ -53,56 +53,56 @@ namespace :navigation_links do
 
   namespace :find_or_create do
     task readinglist: :environment do
-      NavigationLink.where(url: "/readinglist").first_or_create(
-        name: "Reading List",
+      NavigationLink.create_or_update_by_identity(
         url: URL.url("readinglist"),
+        name: "Reading List",
         icon: reading_icon,
         display_only_when_signed_in: true,
-        position: 0,
+        position: 1,
```

Comment:
Just curious: did position change from being 0 based to 1 based?

Reply:
Yes, that's a subtle conversation in some thread I found; namely "Programmers think of staring at 0 but most people won't."  The position shows up in the UI…so here we go.